src/lib/components/table/props.ts

```diff
@@ -51,4 +52,10 @@ export interface QTableProps extends NativeProps, HTMLAttributes<HTMLDivElement>
    * @default false
    */
   dense?: boolean;
+
+  /**
+   * Optionally pass a snippet to render each table cell.
+   * @default undefined
+   */
+  bodyCell?: Snippet<[{ column: QTableColumn; row: QTableRow; style: string }]>;
```

Comment:
Not sure you need to wrap the object in an array in this type, I think only the object would work. Could you test if it works?

Reply:
Tested, that's why I used an array in the first pace. 😄